autogpt/agent/agent.py

```diff
@@ -88,6 +88,7 @@ def start_interaction_loop(self):
                 # Get key press: Prompt the user to press enter to continue or escape
                 # to exit
                 self.user_input = ""
+                logger.typewriter_log("\n") # First log new-line so user can differentiate sections better in console
```

Comment:
Should we use os.linesep rather than \n - does this work on win + linux/mac?

Reply:
This code will also work for Windows.
`os.linesep` is used more when writing to binary files. Not for writing to stdout (as our logger does). The logger uses `print()`.

> Writing code that runs on both Windows and Unix systems always has challenges but thankfully worrying about printing \n versus \r\n isn’t one of them. The print() function is smart enough to convert \n to \r\n when running on Windows.

Source: https://www.idtech.com/blog/what-is-n-in-python